common/cache/lru.go

```diff
@@ -247,12 +257,12 @@ func (c *lru) Release(key interface{}) {
 	entry.refCount--
 }
 
-// Size returns the number of entries currently in the lru, useful if cache is not full
+// Size returns the current size of the lru, useful if cache is not full
```

Comment:
plz update the comment for the Size() method in `cache.go` as well. Plz also mention that the size calculation depends on if the item implements the SizeGetter interface